  File "./groundstation.py", line 319
    usb_dev = finddev(idVendor=0x04d8, idProduct=0xfb31)
                                                       ^
TabError: inconsistent use of tabs and spaces in indentation
./groundstation.py: 3: ./groundstation.py: import: not found
./groundstation.py: 4: ./groundstation.py: import: not found
./groundstation.py: 5: ./groundstation.py: import: not found
./groundstation.py: 6: ./groundstation.py: import: not found
./groundstation.py: 7: ./groundstation.py: import: not found
./groundstation.py: 8: ./groundstation.py: import: not found
./groundstation.py: 9: ./groundstation.py: import: not found
./groundstation.py: 10: ./groundstation.py: import: not found
./groundstation.py: 11: ./groundstation.py: import: not found
./groundstation.py: 12: ./groundstation.py: import: not found
./groundstation.py: 13: ./groundstation.py: import: not found
./groundstation.py: 14: ./groundstation.py: import: not found
./groundstation.py: 15: ./groundstation.py: import: not found
./groundstation.py: 16: ./groundstation.py: import: not found
./groundstation.py: 17: ./groundstation.py: import: not found
./groundstation.py: 18: ./groundstation.py: import: not found
./groundstation.py: 19: ./groundstation.py: import: not found
from: can't read /var/mail/multiprocessing
from: can't read /var/mail/usb.core
./groundstation.py: 24: ./groundstation.py: Syntax error: word unexpected (expecting ")")
./groundstation.py: 3: ./groundstation.py: import: not found
./groundstation.py: 4: ./groundstation.py: import: not found
./groundstation.py: 5: ./groundstation.py: import: not found
./groundstation.py: 6: ./groundstation.py: import: not found
./groundstation.py: 7: ./groundstation.py: import: not found
./groundstation.py: 8: ./groundstation.py: import: not found
./groundstation.py: 9: ./groundstation.py: import: not found
./groundstation.py: 10: ./groundstation.py: import: not found
./groundstation.py: 11: ./groundstation.py: import: not found
./groundstation.py: 12: ./groundstation.py: import: not found
./groundstation.py: 13: ./groundstation.py: import: not found
./groundstation.py: 14: ./groundstation.py: import: not found
./groundstation.py: 15: ./groundstation.py: import: not found
./groundstation.py: 16: ./groundstation.py: import: not found
./groundstation.py: 17: ./groundstation.py: import: not found
./groundstation.py: 18: ./groundstation.py: import: not found
./groundstation.py: 19: ./groundstation.py: import: not found
from: can't read /var/mail/multiprocessing
from: can't read /var/mail/usb.core
./groundstation.py: 24: ./groundstation.py: Syntax error: word unexpected (expecting ")")
